src/telemetry/telemetryProps.ts

```diff
@@ -5,13 +5,17 @@
 
 import type { KnownActiveRevisionsMode } from "@azure/arm-appcontainers";
 import type { SetEnvironmentVariableOption } from "../constants";
+import type { AzdTelemetryProps } from "./AzdTelemetryProps";
 
-export interface DeployWorkspaceProjectTelemetryProps {
+export interface DeployRevisionDraftTelemetryProps extends Pick<AzdTelemetryProps, 'isAzdExtensionInstalled'> {
     revisionMode?: KnownActiveRevisionsMode;
 
-    // Azd
-    isAzdExtensionInstalled?: 'true';
```

Comment:
Careful, looks like this may be getting removed.

Reply:
Oh no, just consolidated under a new interface :)